src/vm/wren_vm.c

```diff
@@ -1070,7 +1083,11 @@ static bool runInterpreter(WrenVM* vm)
       if (fiber->numFrames == 0)
       {
         // If this is the main fiber, we're done.
-        if (fiber->caller == NULL) return true;
+        if (fiber->caller == NULL)
+        {
+          interpretResult.status = WREN_RESULT_SUCCESS;
```

Comment:
> you mean that it could be any Value?

Right.

>  If so, wouldn't AS_CSTRING work?

No, unfortunately. That's only valid to use if you already know the value is an `ObjString*`.